ta/pkcs11/src/pkcs11_token.h

```diff
@@ -144,6 +145,7 @@ struct active_processing {
 	bool always_authen;
 	bool relogged;
 	bool updated;
+	bool oneshotted;
```

Comment:
maybe `bool oneshot_done`?

Reply:
It is there to be a companion for updated so that we know which path has been taken as they cannot be mixed.

oneshot_done would indicate that operation is done and in this particular case that is not true. (eg. you would ask for size of digest.)

Reply:
I think this should not be needed.
Once a oneshot operation is done, the active processing is released. We don't need an intermediate state.

When asking the size, we should not perform the digest operation, just return the requested output size. The digest will be done once C_Digest is call with a suitable output buffer size.

Reply:
Hmm... there is probably a bug also in here when asking for digest size it would digest data two times.

Will add a test case for that to verify it.

Reply:
There wasn't a bug in here -- but added test case just in case.